docs/antctl.md

```diff
@@ -632,3 +633,17 @@ RECORDS-EXPORTED RECORDS-RECEIVED FLOWS EXPORTERS-CONNECTED
 
 For information about Antrea Multi-cluster commands, please refer to the
 [antctl Multi-cluster commands](./multicluster/antctl.md).
+
+### Multicast commands
+
+The `antctl get podmulticaststats [NAME] [-n NAMESPACE]` command can print inbound and outbound multicast statistics for each Pod. Note that IGMP packets are not counted.
```

Comment:
```suggestion
The `antctl get podmulticaststats [NAME] [-n NAMESPACE]` command print inbound and
outbound multicast statistics for each Pod. Note that IGMP packets are not counted.
```
Wrap the line to around 80 or 100 chars,  please check other lines as well.

Reply:
updated